engine/jsb-spine-skeleton.js

```diff
@@ -752,6 +752,7 @@ const cacheManager = require('./jsb-cache-manager');
             if (this._nativeSkeleton) {
                 this._nativeSkeleton.stopSchedule();
```

Comment:
In c++ `SkeletonRenderer::destroy()`, it has already invoked `stopSchedule`, do we need to invoke it in JS here either?